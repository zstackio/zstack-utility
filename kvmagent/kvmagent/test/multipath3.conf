blacklist {

}
defaults {
    user_friendly_names yes
}
devices {
device {
    features 1 queue_if_no_path
    vendor *
}
}
#defaults {
#	polling_interval 	10
#	path_selector		"round-robin 0"
#	path_grouping_policy	multibus
#	uid_attribute		ID_SERIAL
#	prio			alua
#	path_checker		readsector0
#	rr_min_io		100
#	max_fds			8192
#	rr_weight		priorities
#	failback		immediate
#	no_path_retry		fail
#	user_friendly_names	yes
#}
##