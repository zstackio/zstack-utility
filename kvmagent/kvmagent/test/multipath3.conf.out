blacklist {

}
defaults {
    user_friendly_names yes
}
devices {
device {
    vendor *
}
device {
    features 0 
    no_path_retry fail 
    product * 
    vendor *
}

}